vdev/src/commands/version.rs

```diff
@@ -11,8 +11,9 @@ pub(super) struct Cli {}
 impl Cli {
     pub(super) fn exec(self) -> Result<()> {
         app::set_repo_dir()?;
-        let version = env::var("VERSION").or_else(|_| util::read_version())?;
-        let channel = env::var("CHANNEL").or_else(|_| util::release_channel().map(Into::into))?;
+        let mut version = env::var("VERSION").or_else(|_| util::read_version())?;
+        let channel = env::var("CHANNEL")
+            .or_else(|_| env::var("MODE").or_else(|_| util::release_channel().map(Into::into)))?;
```

Comment:
These changes should conflict with what @jonathanpv just merged here.

Reply:
Shucks... ok I'll merge master in. Thanks for the heads up.

Reply:
addressed in https://github.com/vectordotdev/vector/pull/16281/commits/7e3361b4b6e25e163c002fd76548ca7bfa4e8def